src/odemis/acq/stitching/_tiledacq.py

```diff
@@ -598,17 +617,21 @@ def _acquireTiles(self):
             self._moveToTile((ix, iy), prev_idx, self._sfov)
             prev_idx = ix, iy
 
+            if self._focus_points is not None:
+                self._refocus()
+
             das = self._getTileDAs(i, ix, iy)
 
             if i == 0:
                 # Check the FoV is correct using the data, and if not update
                 self._sfov = self._updateFov(das, self._sfov)
 
+            # TODO: why is this step here and not before the acquisition on line 660?
```

Comment:
The idea is that we first acquire at the previous focus, and if we notice the acquisition was too bad, we refocus, and re-acquire. This function here does the "re-focus and re-acquire".